### PR TITLE
UI-Spaces

### PR DESCRIPTION
**Pull request by rolandlor**

### CODE REVIEW

File: src/components/Spaces/Spaces.tsx
Comment (ottpeter) [inline src/components/Spaces/Spaces.tsx:1]: why is there an empty style={{}} ?

**Review by ottpeter — APPROVED** [review]
Started the app, Spaces page looks good, Back text could work as well.
Text could be link as well in NavigationHeader, but this is just an opinion.

File: src/components/NavigationFooter/NavigationFooter.tsx [hunk 1]
```diff
@@ -38,15 +38,15 @@ const NavigationFooter: React.FC = () => {
           <div>Agenda</div>
         </div>
       </Link>
-      <Link to="/rooms">
+      <Link to="/spaces">
```
Comment (bosi95) [inline src/components/NavigationFooter/NavigationFooter.tsx:41]: use ROUTES

File: src/components/NavigationFooter/NavigationFooter.tsx [hunk 2]
```diff
@@ -38,15 +38,15 @@ const NavigationFooter: React.FC = () => {
...
         <div
           className={clsx("navigation-footer__item-container", {
             "navigation-footer__item-container__active":
-              location.pathname === "/rooms",
+              location.pathname === "/spaces",
```
Comment (bosi95) [inline src/components/NavigationFooter/NavigationFooter.tsx:45]: use ROUTES

File: src/components/NavigationFooter/NavigationFooter.tsx [hunk 3]
```diff
@@ -38,15 +38,15 @@ const NavigationFooter: React.FC = () => {
...
           })}
         >
-          <RoomsIcon color={getColor("/rooms")} />
-          <div>Rooms</div>
+          <RoomsIcon color={getColor("/spaces")} />
```
Comment (bosi95) [inline src/components/NavigationFooter/NavigationFooter.tsx:48]: use ROUTES

File: src/pages/SpacesPage/SpacesPage.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import "./SpacesPage.scss";
+import NavigationFooter from "../../components/NavigationFooter/NavigationFooter";
+import NavigationHeader from "../../components/NavigationHeader/NavigationHeader";
+import { CATEGORIES } from "../../utils/constants";
+import SpacesItem from "../../components/Spaces/SpacesItem/SpacesItem";
+
+const SpacesPage: React.FC = () => {
+  return (
+    <div className="spaces">
+      <NavigationHeader to="/home" />
```
Comment (bosi95) [inline src/pages/SpacesPage/SpacesPage.tsx:11]: use ROUTES

**Review by bosi95 — APPROVED** [review]

**Review by Kexort — APPROVED** [review]
Can be merged after the conflict is resolved.
👍

File: src/components/NavigationFooter/NavigationFooter.tsx (hunk 3 above)
Comment (bosi95) [reply src/components/NavigationFooter/NavigationFooter.tsx]: done

File: src/pages/SpacesPage/SpacesPage.tsx (hunk 4 above)
Comment (bosi95) [reply src/pages/SpacesPage/SpacesPage.tsx]: done

File: src/components/NavigationFooter/NavigationFooter.tsx (hunk 2 above)
Comment (bosi95) [reply src/components/NavigationFooter/NavigationFooter.tsx]: done

File: src/components/NavigationFooter/NavigationFooter.tsx (hunk 1 above)
Comment (bosi95) [reply src/components/NavigationFooter/NavigationFooter.tsx]: done

File: src/components/Spaces/Spaces.tsx
Comment (bosi95) [reply src/components/Spaces/Spaces.tsx:1]: done

**Pull request merged**